src/main/resources/openapi/NotificationsAPIs.yaml

```diff
@@ -1,7 +1,7 @@
 openapi: 3.0.0
 info:
   title: Notifications APIs
-  version: '1.17.1'
+  version: '1.17.2'
```

Comment:
As in comment above - worth making sure this gets copied over to the service APIs so there's no errors when the response has the extra fields!